features/nationaliteit.feature

```diff
@@ -29,146 +29,138 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
+      | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
+      | 410                      | 20190604                        |
+      En de persoon heeft nog een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0001                  | 017                   | 20190602                        |
+      Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                              |
       | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 555550001                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '555550001' een nationaliteit met alleen de volgende gegevens
+      | burgerservicenummer | 000000073                           |
+      | fields              | nationaliteiten |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
       | naam                  | waarde   |
       | nationaliteit         | 0001     |
       | redenOpname           | 017      |
       | datumIngangGeldigheid | 20190602 |
-      En heeft de persoon met burgerservicenummer '555550001' GEEN nationaliteit met de volgende gegevens
-      | naam                  | waarde   |
-      | datumIngangGeldigheid | 20190604 |
-      En heeft de persoon met burgerservicenummer '555550001' GEEN nationaliteit met de volgende gegevens
-      | naam                                | waarde |
-      | aanduidingBijzonderNederlanderschap | V      |
-      | redenOpname                         | 310    |
 
     @gba
     Scenario: nationaliteit is onjuist
-      Gegeven de persoon met burgerservicenummer '555550002' heeft de volgende gegevens
-      | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
-      | 1      | 4         | 0263                  | 301                   |                          |                                    |                 | 20100801                        |
-      | 2      | 4         |                       |                       |                          |                                    |                 | 20200727                        |
-      | 2      | 54        | 0100                  | 301                   |                          |                                    | O               | 20200713                        |
+      Gegeven de persoon met burgerservicenummer '000000085' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
+      | 0100                  | 301                   | O               | 20200713                        |
```

Comment:
we voegen de onjuist toe door in de volgende stap te zeggen: "En de 'nationaliteit' is gecorrigeerd naar de volgende gegevens"

Reply:
Aangepast. Moet nog worden geïmplementeerd in de automation code (@MelvLee )